src/Task/RealMeSetupTask.php

```diff
@@ -112,11 +149,12 @@ private function validateInputs($forEnv)
     private function outputMetadataXmlContent($forEnv)
     {
         // Output metadata XML so that it can be sent to RealMe via the agency
-        $this->message(sprintf(
-            "Metadata XML is listed below for the '%s' RealMe environment, this should be sent to the agency so they "
-                . "can pass it on to RealMe Operations staff" . PHP_EOL . PHP_EOL,
+        $this->message(_t(
+            self::class . '.OUPUT_PREFIX',
+            'Metadata XML is listed below for the \'%s\' RealMe environment, this should be sent to the agency so they '
```

Comment:
Please use a `{type}` placeholder or similar and pass it in the third argument as a keyed array